### PR TITLE
DDNet++ support

### PR DESCRIPTION
**Pull request by fokkonaut**

### CODE REVIEW

**Comment by ChillerDragon** [comment]
always use ``git diff --cached`` bro to find those evil spaces before you push o.O

File: src/engine/shared/serverbrowser.cpp [hunk 1]
```diff
@@ -50,6 +50,8 @@ bool IsDDNet(const CServerInfo *pInfo)
 {
 	return str_find_nocase(pInfo->m_aGameType, "ddracenet")
 	    || str_find_nocase(pInfo->m_aGameType, "ddnet")
+	    || str_find_nocase(pInfo->m_aGameType, "ddnet++")
```
Comment (def-) [inline src/engine/shared/serverbrowser.cpp]: wouldn't the strfind before already cover that?

Comment (fokkonaut) [reply src/engine/shared/serverbrowser.cpp]: it doesnt work for me, just with these extra strfinds. also: i dont know why, but for me zooming on bw doesnt work too.

Comment (ChillerDragon) [reply src/engine/shared/serverbrowser.cpp]: oh smart ye probably

Comment (fokkonaut) [reply src/engine/shared/serverbrowser.cpp]: (but the text from the gamemode is blue in the server browser from both gametypes.)

Comment (ChillerDragon) [reply src/engine/shared/serverbrowser.cpp]: @fokkonaut did it work with this fix in the ddnet client?

Comment (fokkonaut) [reply src/engine/shared/serverbrowser.cpp]: https://github.com/ddnet/ddnet/issues/1287

**Pull request closed without merging**